pages/linux/lnav.md

```diff
@@ -0,0 +1,16 @@
+# lnav
+
+> Advanced log file viewer to analyze logs with little to no setup.
+> More information: <https://docs.lnav.org/en/latest/cli.html>.
+
+- View logs of a program, the arguments can be log files, directories or URLs:
+
+`lnav {{path/to/log_directory}}`
+
+- View logs of remote host host1.example.com (SSH passwordless login required):
+
+`lnav {{ssh user@host1.example.com:/var/log/syslog.log}}`
```

Comment:
Is there any good reason to put this in a single token instead of multiple ones? I'd do it like this:

```suggestion
`lnav {{ssh}} {{user}}@{{host1.example.com}}:{{/var/log/syslog.log}}`
```

Reply:
Would `host.example.com` also work? It's just a tiny detail, but still.